linker.go

```diff
@@ -112,6 +112,8 @@ func fixupJumpsAndCalls(insns asm.Instructions) error {
 		}
 
 		switch {
+		case ins.IsLoadOfFunctionPointer() && ins.Constant == -1:
+			fallthrough
```

Comment:
Nit: this is going to return a confusing error `call at %d` when it's really a load. Maybe replace with `%s at %d` and substitute `ins.OpCode`? Then you can also make the function call case fall through to jump resolution.

Reply:
In case of a jump resolution we need to update the `Offset` field instead of `Constant`.
I tried to rewrite it anyway so to avoid repeating the same relocation logic, but I am not sure the result is that better.

Reply:
Sorry i missed that bit, my bad.

Reply:
I ended up amending you commit, I hope you don't mind.

Reply:
Np, that's totally fine :+1: